assets/eip-5058/ERC5058.sol

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
```

Comment:
Please either move the relevant files into the assets folder or remove the OZ reference and let the implementer figure out which impl to use.
```suggestion
```